Multisig/UI/Settings/OwnerKeyManagement/OwnerKeysListViewController/OwnerKeysListViewController.swift

```diff
@@ -140,7 +140,7 @@ class OwnerKeysListViewController: LoadableViewController, UITableViewDelegate,
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let keyInfo = keys[indexPath.row]
-        if App.configuration.toggles.walletConnectOwnerKeyEnabled {
+        if /*App.configuration.toggles.walletConnectOwnerKeyEnabled*/ true {
```

Comment:
I'd rather comment out the branches that are not executing, it's better readable that way instead of `/* ... */`